task1.java

```diff
@@ -0,0 +1,34 @@
+package com.Selenium.homework1;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class task1 {
+    public static void main(String[] args) {
+
+        List<String> l1 = new LinkedList<String>();
+        l1.add("car");
+        l1.add("bus");
+        l1.add("taxi");
+        l1.add("taxi");
+        l1.add("bus");
+        l1.add("bus");
+        l1.add("train");
+        l1.add("taxi");

```

Comment:
You can use addAll() method if you have a lot of elements to put, so it all can be done in one line, e.g.:
l1.addAll(Arrays.asList("car", "bus", "taxi"));

Reply:
fixed in 'task1.changes'